.github/workflows/docker-publish.yml

```diff
@@ -78,14 +77,22 @@ jobs:
   create-multiarch-image:
     runs-on: ubuntu-latest
     name: Create multiarch image
-    needs: [build]
+    needs: [build_for_architecture]
     
     steps:
       - id: repo_name
         uses: ASzc/change-string-case-action@v1
         with:
           string: ${{ github.repository }}
-    
+   
+      # Find the tag to use for versioning code
+      - name: Find latest tag
```

Comment:
This is failing on the public repository as there are no tags (yet).

Please keep in mind that Eclipse Incubator projects are not doing releases, so it would be good idea to either mark it as "technical build" / "Pre-Release" or even better, use non-release-semantics like "build-1234", timestamp or commit hash for tags. You may use the "latest" or "SNAPSHOT" tags like the cloud-connector is doing it.

See https://github.com/eclipse-leda/leda-contrib-cloud-connector/pkgs/container/leda-contrib-cloud-connector%2Fcloudconnector